src/test/java/com/google/graphgeckos/dashboard/DashboardApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.google.graphgeckos.dashboard;
```

Comment:
I believe this file doesn't really test anything. There's no behavior to unit-test as far as I can see. I'd suggest removing this.

Reply:
I included it to create the project barebones structure. I think, there is no need to remove it now because we will be using it not far in the future.

Reply:
Okay. I think my confusion is more about the specific `contextLoads` method:
I'm sure `contextLoads` makes sense as a test in some future PR, but right now there is no "context" that's "loaded" and the method body is empty, so this is just confusing.
How about have an empty `DashboardApplicationTests` class, or at least add a TODO to the `contextLoads` method that this is not its final form?